build/rhel-build-service/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# ARG BUILD_IMAGE=golang:1.11.5
+# ARG BASE_IMAGE=storageos/base-image:0.1.0
+# ARG OPERATOR_IMAGE=storageos/cluster-operator:test
+
+FROM golang:1.11.5 AS build
+# OPERATOR_IMAGE needs to be passed to build/cluster-operator for constructing
+# the ldflags.
+# ARG OPERATOR_IMAGE
+WORKDIR /go/src/github.com/storageos/cluster-operator/
+COPY . /go/src/github.com/storageos/cluster-operator/
+RUN make operator-sdk
+RUN make generate
+# RUN make build/cluster-operator OPERATOR_IMAGE=$OPERATOR_IMAGE
+RUN make build/cluster-operator OPERATOR_IMAGE=registry.connect.redhat.com/storageos/cluster-operator:1.0.0
+RUN make build/upgrader
+
+FROM storageos/base-image:0.1.0
+LABEL name="StorageOS Cluster Operator" \
+      maintainer="support@storageos.com" \
+      vendor="StorageOS" \
+      version="1.0.0" \
+      release="1" \
+      distribution-scope="public" \
+      architecture="x86_64" \
+      url="https://docs.storageos.com" \
+      io.k8s.description="StorageOS Cluster Operator creates, configures and helps manage StorageOS cluster on Kubernetes" \
+      io.k8s.display-name="StorageOS Cluster Operator" \
+      io.openshift.tags="storageos,storage,operator" \
+      summary="Quick and easy StorageOS setup" \
+      description="StorageOS cluster operator deploys and configures StorageOS cluster, and assist with maintenance operations."
```

Comment:
StorageOS transforms commodity server or cloud based disk capacity into \
enterprise-class storage to run persistent workloads such as databases in containers.  Provides \
high availability, low latency persistent block storage.  No other hardware or software is required.